### DISCUSSION TITLE
如何在qt qml 使用这个图表

### DISCUSSION THREAD

**Discussion by ezewu** [antvis/G2#5268] [Q&A]
- [ ] I have searched the [issues](https://github.com/antvis/g2/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
qt里面，一直没有好用的图表，是否可以把这个引入到qt里面

### What does the proposed API look like?
qml里面也有 canvas,应该不用改太多代码

**Comment by hustcc** [comment]
理论上是可以的，G2 的单测中，就是在 node 环境中运行，然后渲染生成图片进行对比。
